### PR TITLE
[Doc Issue][Update code samples in start a conversation][2769142]

### PR DESCRIPTION
**Pull request by v-npaladugu**

### CODE REVIEW

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/resources/bot-v3/bot-conversations/bots-conv-proactive.md](https://github.com/MicrosoftDocs/msteams-docs/blob/153dceea27043503f13cd0ca008752e95331c2c6/msteams-platform%2Fresources%2Fbot-v3%2Fbot-conversations%2Fbots-conv-proactive.md) | 91 | 13 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/3b983cd5-02ef-476b-87a5-cf579166afc0) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/resources/bot-v3/bot-conversations/bots-conv-proactive.md](https://github.com/MicrosoftDocs/msteams-docs/blob/539ce82b8ad725e6a57a7ecd42716825a4f13013/msteams-platform%2Fresources%2Fbot-v3%2Fbot-conversations%2Fbots-conv-proactive.md) | 91 | 13 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/9014b210-1cee-4424-8344-ceade23b60dd) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

File: msteams-platform/resources/bot-v3/bot-conversations/bots-conv-proactive.md [hunk 1]
```diff
@@ -67,13 +67,14 @@ See [Install apps for users](/graph/api/userteamwork-post-installedapps?view=gra
 
 ## Examples
 
-Be sure that you authenticate and have a bearer token before creating a new conversation using the REST API.
+Be sure that you authenticate and have a [bearer token](/azure/bot-service/rest-api/bot-framework-rest-connector-authentication    &tabs=multitenant#bot-to-connector) before creating a new conversation using the REST API.
 
 ```json
```
Comment (Wajeed-msft) [inline msteams-platform/resources/bot-v3/bot-conversations/bots-conv-proactive.md:72]: Could you please update this with separate HTTP and JSON sections?

**Review by Wajeed-msft — DISMISSED** [review]

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/resources/bot-v3/bot-conversations/bots-conv-proactive.md](https://github.com/MicrosoftDocs/msteams-docs/blob/0da2dc17f6042b0077d959a2b153dd071d2142fa/msteams-platform%2Fresources%2Fbot-v3%2Fbot-conversations%2Fbots-conv-proactive.md) | 91 | 13 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/ed3a7f2d-56d6-47e3-bb1f-19c6a1d2a90c) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/resources/bot-v3/bot-conversations/bots-conv-proactive.md](https://github.com/MicrosoftDocs/msteams-docs/blob/d63eac1515f0ae639008c0652fe3321e023b0988/msteams-platform%2Fresources%2Fbot-v3%2Fbot-conversations%2Fbots-conv-proactive.md) | 91 | 13 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/eaebca94-bef0-47e7-884d-03bfb09455b9) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/resources/bot-v3/bot-conversations/bots-conv-proactive.md](https://github.com/MicrosoftDocs/msteams-docs/blob/5a6d3a66eeeeeb6fcacbe8ad253628b894e096d9/msteams-platform%2Fresources%2Fbot-v3%2Fbot-conversations%2Fbots-conv-proactive.md) | 91 | 13 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/9a1cd59e-b3f1-48ec-a186-2e4a565995a2) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/resources/bot-v3/bot-conversations/bots-conv-proactive.md](https://github.com/MicrosoftDocs/msteams-docs/blob/a89328774082e85520463d9fb98fbc145b52f161/msteams-platform%2Fresources%2Fbot-v3%2Fbot-conversations%2Fbots-conv-proactive.md) | 91 | 13 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/9aa5c659-df33-4ea2-82c3-c2235bcc6b72) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Pull request merged**